OpenRA.Mods.RA/Traits/SpawnActorOnDeath.cs

```diff
@@ -42,6 +42,9 @@ public class SpawnActorOnDeathInfo : ITraitInfo
 		[Desc("Skips the husk actor's make animations if true.")]
 		public readonly bool SkipMakeAnimations = true;
 
+		[Desc("Should no husk been spawned when the 'Creeps' setting is false?")]
```

Comment:
Can you please fix all these references to 'husks'?  They shouldn't have survived the initial renaming to `SpawnActorsOnDeath`.